packages/react/src/components/PhoneNumberInput/PhoneNumberInput.tsx

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {FlagOutlined} from '@mui/icons-material';
+import Select, {SelectChangeEvent, SelectProps as MuiSelectProps} from '@mui/material/Select';
+import clsx from 'clsx';
+import {
+  ChangeEvent,
+  forwardRef,
+  ForwardRefExoticComponent,
+  MutableRefObject,
+  ReactElement,
+  useEffect,
+  useState,
+} from 'react';
+import Flag from 'react-world-flags';
+import {countries, Country} from './constants';
+import {WithWrapperProps} from '../../models';
+import {composeComponentDisplayName} from '../../utils';
+import Box, {BoxProps} from '../Box';
+import InputLabel from '../InputLabel';
+import ListItemIcon from '../ListItemIcon';
+import MenuItem from '../MenuItem';
+import './phone-number-input.scss';
+import OutlinedInput, {OutlinedInputProps as MuiOutlinedInputProps} from '../OutlinedInput';
+import Typography from '../Typography';
+
+export interface PhoneNumberInputProps extends BoxProps {
+  OutlinedInputProps?: Omit<MuiOutlinedInputProps, 'id' | 'label' | 'placeholder' | 'value' | 'type'>;
```

Comment:
Lets document the interface.

Reply:
Fixed https://github.com/wso2/oxygen-ui/pull/64/commits/481bd8618800338377fcff1956b262066106c9c5